test/FunctionalTests/Tests/DataEdmLib/FunctionalUtilities/NavigationTestModelBuilder.cs

```diff
@@ -1412,9 +1396,9 @@ public static IEdmModel RecursiveThreeContainmentNavigationsWithEntitySet()
 
             personSet.AddNavigationTarget(personToPet, petSet);
             petSet.AddNavigationTarget(personToPet.Partner, personSet);
-            petSet.AddNavigationTarget(petToHome, homeSet);
+//            petSet.AddNavigationTarget(petToHome, homeSet);

```

Comment:
if we don't want to remove these, it's better to have some comments?

Reply:
I will create a separate pull request to remove these.